app/import_test.go

```diff
@@ -204,12 +204,26 @@ func TestImportBulkImport(t *testing.T) {
 	})
 
 	t.Run("First item after version without type", func(t *testing.T) {
-		data := `{"type": "version", "version": 1}
```

Comment:
nit 0/5 this rename seems redundant and just adds noise to the PR :)

Reply:
I was really uncertain about adding it in the first place. Happy to remove it. :)

Reply:
reverted the changes here :)